arionBackend/api/query/query.py

```diff
@@ -54,3 +54,16 @@ def get(self, request, query_id, format=None):
 		except ObjectDoesNotExist:
 			return HttpResponse(status=status.HTTP_404_NOT_FOUND)
 		return JsonResponse(query.to_json(), safe=False)
+
+
+class GetQueriesByEventTypeId(GetXByIdView):

```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Add a docstring to this class. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.tommartensen.de/coding_rules#rule_key=python%3AS1720)